README.md

```diff
@@ -71,6 +71,189 @@ const result = validate(value, validations)
 | null          |                   |                   |  
 | undefined     |                   |                   |  
 
+
+
+#### Contains
+
+`contains` -  The contains validator is useful for validating allowance or restriction in certain values.
+It checks that the given value exists in the target given by the **allowed** or  **notAllowed** option.
+
+You can specify the validator as a list, string or as an object (in which case the keys of the object are used).
+
+**allowed option examples:**
+
+```javascript
+var list = ["small", "medium", "large"]
```

Comment:
I don't think it is necessary to have a example with valid values. For simplicity I would leave the invalid values only

Reply:
I made the modification and added just one example of success and the rest mounting only failures